### PR TITLE
Add option to disable automatic PR creation

### LINKED ISSUE #79: Option to disable automatic creation of PR
Hi there!  Amazing work on this tool 👍.  I was wondering if there's an option to not create the PR but to just push the changes to your local fork of a repository and open the compare page.  The first time I ran the tool, it only opened the GitHub compare page in my browser, which is exactly what I wanted.  The second time, though, it automatically created the pull request.  An option to just create the branch and push the changes would be great.

Thanks, Madhav.

### PR DESCRIPTION
**Pull request by madhavarshney**
Closes #79.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -323,13 +327,17 @@ async function go() {
     console.log(finalDiff);
     console.log();
 
+    const createPR = !(noPR || await findGithubFile('PULL_REQUEST_TEMPLATE'));
+
     await respondToUserInput(
-      'Are you sure you want to create a pull request with these corrections?',
+      `Are you sure you want to ${createPR ? 'create a pull request with' : 'push and send'} these corrections?`,
```
Comment (madhavarshney) [inline index.js]: Any ideas for better wording?

File: index.js [hunk 2]
```diff
@@ -68,6 +68,9 @@ const optionDefinitions = [
   {
     name: 'repository', alias: 'r', typeLabel: '<username/repository or URL>', description: 'The repository to spellcheck.',
   },
+  {
+    name: 'no-pr', type: Boolean, description: 'Do not create the pull request.',
```
Comment (tbroadley) [inline index.js]: How about "Do not automatically create a pull request on the target repository."?

File: index.js (hunk 1 above)
Comment (tbroadley) [reply index.js]: How about:

1. "Are you sure you want to create a pull request with these corrections?" if `createPR` is true
2. "Are you sure you want to push these changes to <branch name> on <repository/fork owner and name>?" if `createPR` is false

File: index.js [hunk 3]
```diff
@@ -323,13 +327,17 @@ async function go() {
...
-      'Are you sure you want to create a pull request with these corrections?',
+      `Are you sure you want to ${createPR ? 'create a pull request with' : 'push and send'} these corrections?`,
       [
         {
           command: 'y',
           meaning: 'yes',
-          description: 'Create a pull request with the specified changes.',
+          description: createPR
+            ? 'Create a pull request with the specified changes.'
+            : `Push the changes${!quiet ? ' and open the pull request creation page' : ''}.`,
```
Comment (tbroadley) [inline index.js]: :+1:

File: index.js [hunk 4]
```diff
@@ -349,7 +357,7 @@ async function go() {
             console.log('Pushing to remote "origin"...');
             await git(clonePath, `push ${repoFullUrl} refs/heads/${branchName}`);
 
-            if (await findGithubFile('PULL_REQUEST_TEMPLATE') && !quiet) {
+            if (!quiet && !createPR) {
```
Comment (tbroadley) [inline index.js]: If the user sets the `--no-pr` and `--quiet` flags, `quiet === true` so a PR will be created. That's not the behaviour I would expect.

Comment (madhavarshney) [reply index.js]: Oops...

**Comment by madhavarshney** [comment]
@tbroadley - I updated the PR. Please take a look.

**Comment by tbroadley** [comment]
Thanks, I'll take a look soon! Hopefully in the next 24 hours.

**Comment by madhavarshney** [comment]
@tbroadley - Sorry for disturbing you, but is there anything I need to do?

**Pull request merged**